### PR TITLE
Database connector for AWS RDS

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**

### CODE REVIEW

File: content/en/docs/partners/aws/amazon-rds-database-connector.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+To configure the Mendix Database Connector for Amazon RDS, follow these steps: 
+
+1. Create a database in Amazon RDS by doing the following steps:
+    1. Select **PostgreSQL** as the database type.
+    2. Select the required template.
+    3. Enter a database instance identifier. 
+    4. Create the **Master Username** and **Master Password**. 
+    
+        {{% alert color="info" %}}
+        Store the username and password in a safe location.
```
Comment (MarkvanMents) [inline content/en/docs/partners/aws/amazon-rds-database-connector.md]: This looks wrong when built.
For formatting this has to either be on the same line as the alert tag, or start in column 1.
It's a weirdness in markdown/Hugo.

File: content/en/docs/partners/aws/amazon-rds-database-connector.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+4. Configure the Database module to [connect to AWS RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ConnectToPostgreSQLInstance.html#USER_ConnectToPostgreSQLInstance.JDBCDriverPostgreSQL) by doing the following steps:
+    1. Use one of the microflow actions provided in the Database connector in a microflow, as described in [Database](/appstore/connectors/database-connector/).
+        For example, add the *ExecuteStatement* action.
+    2. In the AWS console, in the RDS Database dashboard, find and copy the **Connectivity & Security** endpoint.
+    3. In the microflow that you created, in the *ExecuteStatement* microflow action, configure the **jdvcUrl** parameter in the following way: `jdbc:postgresql://{endpoint copied from the AWS console}:5432/`
+    4. In the **User name** and **Password** parameters, enter the Master Username and Master Password that you configured in AWS RDS in step 1d above, or other credentials with equivalent access rights for AWS RDS. 
+    5. In the **Sql** parameter, enter your SQL statement.
+        For example, to create a table, you can use the following statement:
+
+            ```text {linenos=false}
```
Comment (MarkvanMents) [inline content/en/docs/partners/aws/amazon-rds-database-connector.md]: The linter picked this up as having the wrong indentation (one too far to the right).
Also, sql is a valid language tag, so you could use this to get automatic highlighting.

File: content/en/docs/partners/aws/amazon-rds-database-connector.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            CREATE TABLE employee (
+                emp_id INT PRIMARY KEY,
+                first_name VARCHAR(50) NOT NULL,
+                last_name VARCHAR(50) NOT NULL)
+            ```
+
+        {{% alert color="info" %}}
```
Comment (MarkvanMents) [inline content/en/docs/partners/aws/amazon-rds-database-connector.md]: This has the same issue as the one mentioned above when built. Indented alert blocks are a pain until you get used to them.

File: content/en/docs/partners/aws/amazon-rds-database-connector.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+   
+    5. In the **Connectivity** section, set **Public access** to **yes**.
+    6. Make sure that **Database Authentication** is set to **Password authentication**.
+2. Optional: If you want to limit access to the database to specific apps, do the following steps:
+    1. After the database is created, access the [security group rules](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html#SecurityGroupRules).
+    2. Select the first security group.
+    3. Click **Edit inbound rules**.
+    4. Click **Add rule**.
+    5. Select **PostgreSQL** as the database type.
+    6. Select the IP address of your Mendix app as the source.
```
Comment (MarkvanMents) [inline content/en/docs/partners/aws/amazon-rds-database-connector.md]: Can you just check on this with the devs? I'm not sure that you can identify a single IP address for an app running on Mendix Cloud - you may have to whitelist all the IP addresses mentioned here: https://docs.mendix.com/developerportal/deploy/mendix-ip-addresses/#22-outgoing-ip (I'm pretty sure it is the outgoing one you are interested in). You may even want to link to this doc?

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi Kasia,
Content is great.
I just happened to look at it because I wanted to see whether our linter (which Natasa is in the process of automating) would complain about the embedded lists not having spaces above and below them (it does complain about the top level lists if they have no spaces!). It doesn't and Hugo parses them correctly, so it isn't an issue.

You probably haven't finished reviewing this and would have picked up what is in my comments anyway, but since I was looking at the file I picked up a couple of things in the formatting. The linter was helpful with the code block, but doesn't help with the alert formatting unfortunately. I'll talk to Natasa about whether this is something we can add to the linting.

I also added a note about IP addresses which you might want to think about. Let me know if you want to discuss.

Content-wise, the PR looks great.

File: content/en/docs/partners/aws/amazon-rds-database-connector.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/partners/aws/amazon-rds-database-connector.md]: Done.

File: content/en/docs/partners/aws/amazon-rds-database-connector.md (hunk 1 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/partners/aws/amazon-rds-database-connector.md]: Done.

File: content/en/docs/partners/aws/amazon-rds-database-connector.md (hunk 3 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/partners/aws/amazon-rds-database-connector.md]: Done.

File: content/en/docs/partners/aws/amazon-rds-database-connector.md (hunk 4 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/partners/aws/amazon-rds-database-connector.md]: Done.

**Pull request merged**